internal/backend/memory.go

```diff
@@ -113,6 +113,18 @@ func findObject(obj Object, objectList []Object, matchGeneration bool) int {
 	return -1
 }
 
+// findObject looks for an object in the given list and return the index where it
```

Comment:
nit: make the comment match the method name
```suggestion
// findLastObjectGeneration looks for an object in the given list and return the index where it
```